docs/API-functions.md

```diff
@@ -544,11 +545,29 @@ type with the option value to be set.
 
 -Errors:
 
-  * `SRT_EINVSOCK`: Socket `u` indicates no valid socket ID
+  * `MN_INVAL`: Socket `u` indicates no valid socket ID
```

Comment:
`MN_INVAL` is not an SRT error code. Maybe you meant `SRT_EINVPARAM`?

Reply:
Yes, that was a mistake.

Reply:
Moreover, I believe `SRT_EINVSOCK` is exactly what is returned in case of invalid socket supplied to the function.

Still, `SRT_EINVPARAM` should be added here, as it is reported in case of invalid condition under which this function is called, inability of performing the operation, or an internal error.